frontend_lib/src/component/Timeline/Comment.styl

```diff
@@ -1,4 +1,5 @@
 @import '../../css/Variable.styl'
+linkColor = #007bff // This color comes from bootstrap
```

Comment:
Can you add this variable in Variable.styl instead?

and also move the one in `frontend/src/component/FeedItem/Preview.styl`

Reply:
fixed